src/Modules/Profie/index.js

```diff
@@ -31,10 +31,11 @@ class Profile extends Component {
     if (currentUserName !== previousUserName) this.fetchData();
   }
   // handler for editing the user profile details
-  handleEditProfile = (e) => {
```

Comment:
The function name doesn't suit what it is doing, is it a single function for reset the form and starting/closing edit?